opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -2087,7 +2087,8 @@ namespace Opm
     StandardWell<TypeTag>::
     getWellConvergence(const WellState& well_state,
                        const std::vector<double>& B_avg,
-                       Opm::DeferredLogger& deferred_logger) const
+                       Opm::DeferredLogger& deferred_logger,
+                       const bool relax_tolerance) const
```

Comment:
Should be marked as unused argument to avoid warnings. (Or implement the relaxation?)